Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.css

```diff
@@ -0,0 +1,87 @@
+.details-section > .content > .group > .row.font-variation {
+    display: grid;
+    gap: 6px;
+    grid-template-columns: 36px 1fr minmax(max-content, 36px);
+    padding-block: 5px;
+    padding-inline: 20px;
+}
+
+.details-section > .content > .group > .row.font-variation > .label:not(:empty) {
+    grid-column: 1 / -1; /* The variation axis name, if present, spans the full width of the grid container. */
+}
+
+.details-section > .content > .group > .row.font-variation > .tag {
+    grid-column: 1;
+    color: var(--text-color-secondary);
+}
+
+.details-section > .content > .group > .row.font-variation > .value {
+    grid-column: 3;
+    text-align: end;
+    font-variant-numeric: tabular-nums;
+}
+
+.details-section > .content > .group > .row.font-variation > .variation-range {
+    display: grid;
+    grid-column: 2;
+    grid-template-columns: 1fr 1fr;
+}
+
+.details-section > .content > .group > .row.font-variation > .variation-range > input[type="range"] {
```

Comment:
Part of me wants to say this ranged slider control should be a separate JS class and CSS file, but this is probably fine for now.

Reply:
I'll keep it in this file for now since it's a specialized input. 
When we need it for something else, we can extract it then.